packages/calling/src/CallingClient/CallingClient.ts

```diff
@@ -3,7 +3,7 @@
 /* eslint-disable @typescript-eslint/no-shadow */
 import * as Media from '@webex/internal-media-core';
 import {Mutex} from 'async-mutex';
-import {filterMobiusUris, handleRegistrationErrors, validateServiceData} from '../common/Utils';
+import {filterMobiusUris, handleCallingClientErrors, validateServiceData} from '../common/Utils';
```

Comment:
No testcase covers the scenario where there was a registration reconnect during active calls.
Following lines to be covered in a testcase.
Method at Line 351: ` private callsClearedHandler = async ()`

Reply:
It will be covered as part of call related ticket on which Shreyas is currently working on.

Reply:
Thanks!